cocotb/scheduler.py

```diff
@@ -49,7 +49,7 @@
 # Debug mode controlled by environment variables
 if "COCOTB_ENABLE_PROFILING" in os.environ:
     import cProfile, StringIO, pstats
```

Comment:
I'm actually confused as to why import `StringIO` from `io` in `result.py` but import the module directly here.

Assume this is something to do with Python3 compatibility?